docker/scripts/load_data.py

```diff
@@ -274,8 +289,9 @@ def build_record(row):
                     ld = LoadDatasets(API_KEY)
                     # TODO(@frascuchon): Remove these datasets creation
                     ld.load_mock_dataset_for_filter_metadata()
```

Comment:
I think we can remove also this one since we have a real one with metadata properties
```suggestion
```